configs/cantaloupe.properties-latest.tmpl

```diff
@@ -0,0 +1,377 @@
+###########################################################################
+# cantaloupe.properties configuration
+# For Cantaloupe v4.x
+# Created using https://github.com/medusa-project/cantaloupe/blob/release/4.0/cantaloupe.properties.sample
+###########################################################################
```

Comment:
@cachemeoutside I thought you said the latest was 5.x now?

Reply:
I didn't account for 5.x configurations in this PR. The ticket was specifically for 4.1 as stable. Should I bundle 5.x changes here?

Reply:
Yeah, I really should have caught it at https://github.com/UCLALibrary/docker-cantaloupe/commit/a0ae76e36c9aa204b52a202b34674017a0074530 where we bumped the dev build up to 5.x, but since I missed it then it would be good to handle now.

"We bumped it up" == "We dealt with a HEAD build now being 5.x"